mapbox_maps_flutter_mobile/LICENSE

```diff
@@ -75,31 +75,6 @@ THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
 OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 ```
 
```

Comment:
Is removing this intentional?

Reply:
yes, turf has been moved to mapbox_maps_flutter_interface, however, before publishing these new packages we need to discuss which one should be included in the license file.